site/src/pages/404Page/404Page.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import NotFoundPage from "./404Page";
+
+const meta: Meta<typeof NotFoundPage> = {
+  title: "components/NotFoundPage",
+  component: NotFoundPage,
+};
+
+export default meta;
+type Story = StoryObj<typeof NotFoundPage>;
+
+export const Default: Story = {};
```

Comment:
Storybook likes when single story files export one with the name of the "title", and collapses it in the UI nicely. :)

We do this in several other stories files:

```suggestion
const Example: Story = {};
export { Example as NotFoundPage };
```